data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/command/dataexport/ExportCommand.java

```diff
@@ -64,17 +64,18 @@ public Integer call() throws Exception {
           spec.commandLine(), dataChunkSize, DataLoaderError.INVALID_DATA_CHUNK_SIZE);
       validatePositiveValue(spec.commandLine(), maxThreads, DataLoaderError.INVALID_MAX_THREADS);
 
-      StorageFactory storageFactory = StorageFactory.create(scalarDbPropertiesFilePath);
       TransactionFactory transactionFactory = TransactionFactory.create(scalarDbPropertiesFilePath);
-      TableMetadataService metaDataService =
-          new TableMetadataService(storageFactory.getStorageAdmin());
+      TableMetadata tableMetadata;
+      try (DistributedTransactionAdmin admin = transactionFactory.getTransactionAdmin()) {
```

Comment:
So previously, we didn’t close the storage admin instance, right? If that’s the case, could you please check if the same issue exists elsewhere as well?